### DISCUSSION TITLE
is there a way to disable PR creation?

### DISCUSSION THREAD

**Discussion by pere3** [renovatebot/renovate#15484] [Request Help]
Hello, renovate team! Thanks for a great tool!

I am currently tring to migrate our dependency upgrade process from dependabot to renovate, as a result of git backend switch to gitea. It was very nice to see a native gitea support in renovate! Thanks again.

But i stumbled upon a difficulty of finding any configuration option for disabling PR creation in process of renovation. Is there even an option for this?
Our internal CI is able to maintain all PR logic by itself, and all i want renovate to do - is to create a branch (name of which is a Jira ticket number, and it will be given through branchName option) and commit all dependency updates to it.

Is there even a way to go for me?

**Comment by rarkins** [comment]
There might be a hack/trick to suppressing PRs, but I'm not sure any are good. For example if you set prCreation=success and then intentionally had a failed check, that should do it.

If you'd like to submit a PR to support prCreation=never, we would accept it.

**Reply by pere3** [reply]
Thanks for a quick response, i'll try to look into it!

**Comment by pere3** [comment]
I would also like to mention that in a case i presented here (internal CI with 1-to-1 match between Jira issue and repository branch) - i'm going to use **branchName** directive https://docs.renovatebot.com/configuration-options/#branchname

There is a comment suggesting "a better approach you should use", so i'm wonder is there really? Or in that particular case i can safely use branchName and be ok with that?

**Reply by rarkins** [reply]
Can you give an example of how you'd use it? We recommend you normally don't change it directly and instead change 1-2 of its components if you must

**Reply by pere3** [reply]
we are going to use manual renovate runs, with repo maintainers controls when to run renovate and on which repo

1. jira issue created upon starting renovate run pipeline
2. given issue name used as a branchName and groupName (to combine all updates under one commit and PR) parameters for renovate run 
3. renovate does its job, bumping all found dependencies and doing single commit (it also does create PR, but we a going to left aside this functionality, as it doesn't break our flow)
4. internal CI is triggered to process Jira Issue, which corresponds to finding an branch by itself, creating a PR and starting a CI/CD process

**Reply by rarkins** [reply]
Sounds like you're running it in a very custom way. I guess "use at your own risk" applies, for now manually setting the branch name should work. 

You're welcome to submit a PR to implement prCreation=never functionality if you need it

**Reply by pere3** [reply]
thank you so much for all the responses and thoughts! renovate community feels very welcoming and cozy with all that

i'm mostly do python, but will consider a bit of a dive into node

**Comment by Smixi** [comment]
Hi ! I'm interessted by having only the dependency dashboard. I may try to make a PR for this (I'm a new contributor).

In my use case, it's only to track dependencies, and so I would not even want to create a branch. prCreation == never feels like the option will create the branch but not the PR, or is it clear enough ?
Also, branchConcurrentLimit and prConcurrentLimit cannot be set to 0 to not create branch.

If I understand things correctly, settings an option flag to the `updateRepo(config, branches)` [here](https://github.com/renovatebot/renovate/blob/5c3eade7d2c2dd44f309272892b1dd18f03c0fb5/lib/workers/repository/index.ts#L77) should be enough with the new option ?

**Reply by rarkins** [reply]
You just need to set dependencyDashboardApproval=true

**Reply by Smixi** [reply]
Oh ! I completly missed that. Thank you a lot !
